task-sdk/src/airflow/sdk/execution_time/supervisor.py

```diff
@@ -1272,7 +1291,6 @@ class InProcessTestSupervisor(ActivitySubprocess):
     """A supervisor that runs tasks in-process for easier testing."""
 
     comms: InProcessSupervisorComms = attrs.field(init=False)
-    stdin = attrs.field(init=False)
```

Comment:
@kaxil I assume `dag.test()` is covered by unit tests right? I'm 80% sure this change is safe. Ditto for the removal of lock on L`1246`

Reply:
They are, and this fails without this